contacts.py

```diff
@@ -555,54 +555,57 @@ def send_status(self, data, args):
         # These are fields allowed in the send_status, and just copied from top level into each data point
         # Note memo is not supported yet and is a placeholder
         repeated_fields = {k: data.get(k) for k in ['memo', 'replaces', 'status'] if data.get(k)}
-        return self.send_or_sync(data, repeated_fields)
+        self.send_or_sync(data, repeated_fields)
+        return {"status": "ok"}
 
     # Common part of both /status/send and sync reception
-    def send_or_sync(self, data, repeated_fields):
-        floating_seconds = current_time()
+    def send_or_sync(self, data, repeated_fields=None, floating_seconds=None):
+        if not floating_seconds:
```

Comment:
why this if?  why not just set default of floating_seconds to current_time() in param list?

Reply:
DIfferent problem I'm not sure if  `floating_seconds=current_time()` always executes the `current_time()` even if floating_seconds is set and the result isn't assigned.  Could be wrong about this, but it was safe not to use it. .... if you are sure it doesnt get executed I can change this.

Reply:
i doubt that it does, but you can certainly test it.